### DISCUSSION TITLE
Dependency Track for CI/CD

### DISCUSSION THREAD

**Discussion by bhuvi11** [DependencyTrack/dependency-track#1262] [Q&A] [answered]
Hello Team,

I am looking for options to integrate dependency track as part of CICD pipeline.
Is there a way i can integrate using APIs? I understand we need to get the API key to access APIs, but in CICD how can i use the v1/teams - api call to create a new team and API without authentication?

**Comment by stevespringett** [comment, marked as answer]
See:
https://docs.dependencytrack.org/usage/cicd/
https://docs.dependencytrack.org/integrations/community-integrations/
https://docs.dependencytrack.org/integrations/rest-api/

> API without authentication

DT does not support that. All requests, with the exception of a few documented cases, require authentication and authorization.

Your CI/CD pipeline simply needs to properly manage the secrets (the DT API key) necessary for integration.
